wkcuber/Dockerfile

```diff
@@ -18,6 +18,12 @@ COPY webknossos/poetry.lock /webknossos/
 COPY webknossos/pyproject.toml /webknossos/
 COPY webknossos/README.md /webknossos/
 
+RUN mkdir /cluster_tools
+COPY cluster_tools/cluster_tools /cluster_tools/cluster_tools
+COPY cluster_tools/poetry.lock /cluster_tools/
+COPY cluster_tools/pyproject.toml /cluster_tools/
+COPY cluster_tools/README.md /cluster_tools/
+
```

Comment:
Were they linked instead before? I guess this was simply missing, or did something else make this necessary?

Reply:
Earlier versions simply required them as a package via poetry. Now that it has changed to `cluster_tools = { path = "../cluster_tools/", develop = true }`, it has to be copied into the docker image. This is analogous to how webknossos is added to the docker image.